lib/activesupport/all/activesupport.rbi

```diff
@@ -323,12 +331,17 @@ class Array
 
   sig { params(elements: T.untyped).returns(T.untyped) }
   def without(*elements); end
+
+  sig { params(object: T.untyped).returns(T::Array[T.untyped]) }
+  def self.wrap(object); end
 end
 
 module ActiveSupport::NumberHelper
+  extend(::ActiveSupport::NumberHelper)
+
   sig do
     params(
-      number: T.any(Integer, Float, String),
+      number: T.any(Numeric, String),
```

Comment:
These (and many other sigs) were typed as `T.any(Integer, Float)` but I confirmed any `Numeric` (BigDecimal, Rational) would work.